src/data-dictionary/events/Span/error.type.md

```diff
@@ -0,0 +1,8 @@
+---
+name: error.type
+type: attribute
+events:
+  - Span
+---
+
+Either HTTPError or NetworkFailure, depending on whether the error is a result of a failed request to a host or a failure on the cellular network.
```

Comment:
Should HTTPError and Networkfailure be in code?

Reply:
@urbiz-nr, I added the back ticks for code.